### ISSUE TITLE
about the problem that train and val set has the same tid_curr.

### ISSUE THREAD

**Issue by hyeseung-of-dable** [ifzhang/FairMOT#400]
I have a question about gen_labels_crowd_id.py.

```
gen_labels_crowd(data_train, label_train, ann_train)
gen_labels_crowd(data_val, label_val, ann_val)
```
In this code, labels_with_id\train\*.txt and labels_with_id\val\*.txt is generated seperately. 
So if there are 1000 pedestrians in the training set and 100 pedestrians in the validation set, 
0~99th pedestrian will have the same tid_curr between the train and val txt file despite being different people.

I wonder if this doesn't matter.

**Comment by danishmansari** [comment]
Train and Val set is used at different and separate stages of training(train/eval), so it should not matter.
